src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Startup.cs

```diff
@@ -65,24 +66,24 @@ public void ConfigureServices(IServiceCollection services)
     #else
                 options.UseSqlite(
                     Configuration.GetConnectionString("DefaultConnection"),
-                    sqlOptions => sqlOptions.MigrationsAssembly("Company.WebApplication1")));
+                    sqlOptions => sqlOptions.MigrationsAssembly("Company.WebApplication1")
+                ));
     #endif
-
             services.AddIdentity<IdentityUser, IdentityRole>(options => options.Stores.MaxLengthForKeys = 128)
                 .AddEntityFrameworkStores<IdentityDbContext>()
                 .AddDefaultUI()
                 .AddDefaultTokenProviders();
 
-#elseif (OrganizationalAuth || IndividualB2CAuth)
+#elif (OrganizationalAuth || IndividualB2CAuth)
```

Comment:
Were these just.... always wrong??

Reply:
After talking to @seancpeters we can change these to #elif going forward. @Eilon I'll keep this for dev